### PR TITLE
Enhancement welcome pages header

### PR DESCRIPTION
**Pull request by capitalsav**
#44 Issue
Made header for welcome pages.
Only standart header (for first displayed page).
Made markup for desktop and mobile devices.
Made navbar sidebar for mobile devices.

### CODE REVIEW

File: app/javascript/packs/header.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+$(document).ready(function(){
+    $( "#header-collapse-button" ).click(function() {
```
Comment (polyakovigor) [inline app/javascript/packs/header.js]: use `.on('click'`

File: app/javascript/style/fonts.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+@font-face {
+  font-family: 'fonts/PT Sans';
+  font-style: normal;
+  //src: local('fonts/PT Sans Web Regular'), local('fonts/PT_Sans-Web-Regular'),
```
Comment (polyakovigor) [inline app/javascript/style/fonts.scss]: Why did not you delete this line?

**Review by polyakovigor — COMMENTED** [review]
I think we need to create scss file with colors

**Comment by capitalsav** [comment]
@polyakovigor 
Did you mean that we should use css variables in separated file to contain colors.

**Comment by polyakovigor** [comment]
@Capitalsav Yes.

**Review by polyakovigor — APPROVED** [review]

**Review by vladfreel — APPROVED** [review]

**Pull request merged**